--- a/html-src/gen-html.py
+++ b/html-src/gen-html.py
@@ -4,12 +4,15 @@
 import os
 import sys
 
+pysollib_dir = '../'
+pysollib_path = os.path.join(sys.path[0], pysollib_dir)
+sys.path[0] = os.path.normpath(pysollib_path)
+
 from pysollib.gamedb import GAME_DB
 from pysollib.gamedb import GI
 from pysollib.mfxutil import latin1_normalize
 from pysollib.mygettext import fix_gettext
 # outdir = '../html'
-pysollib_dir = '../'
 
 
 builtins._ = lambda x: x
@@ -37,10 +40,6 @@
     return ret
 
 
-pysollib_path = os.path.join(sys.path[0], pysollib_dir)
-sys.path[0] = os.path.normpath(pysollib_path)
-# print sys.path
-
 fix_gettext()
 
 files = [
