--- a/pysollib/pysolrandom.py	2019-01-31 17:22:56.667717088 +0100
+++ b/pysollib/pysolrandom.py	2019-01-31 17:23:13.535588891 +0100
@@ -24,15 +24,10 @@
 
 # imports
 import re
+import sys
 import time
 from pysollib.mfxutil import SubclassResponsibility
-try:
-    import random2
-except ImportError:
-    raise ImportError(
-        "You need to install " +
-        "https://pypi.python.org/pypi/random2 using pip or similar.")
-
+import random
 
 # ************************************************************************
 # * Abstract class for PySol Random number generator.
@@ -97,13 +92,15 @@
 # * uses the standard python module `random'
 # ************************************************************************
 
-class MTRandom(BasicRandom, random2.Random):
+class MTRandom(BasicRandom, random.Random):
 
     def __init__(self, seed=None):
         if seed is None:
             seed = self._getRandomSeed()
         BasicRandom.__init__(self)
-        random2.Random.__init__(self, seed)
+        random.Random.__init__(self, seed)
+        if sys.version_info > (3,):
+            self.seed(a=seed, version=1)
         self.initial_seed = seed
         self.initial_state = self.getstate()
         self.origin = self.ORIGIN_UNKNOWN
